### PR TITLE
Validations and specs

### PR DESCRIPTION
**Pull request by IsmaelMastronardi**
## In this branch, I worked on adding validation to the previously created models: 
- User checks that the name is not blank
- Post checks that the title is not blank or longer than 250 characters, 
- Post checks that the comments_counter and likes_counter are integers greater than 0.

### User
- User with no name
- User with a valid name
- Users top_posts method
- User post_count 
### Post
- Post with no title
- Post with more than 250 characters
- Post with valid Title
- Post comments_counter and likes_counter

### CODE REVIEW

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi @IsmaelMastronardi  🙋‍♀️,  

While you made a great effort on this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
 
You have submitted a project that does not have a professional `Getting Started` section. The `Getting Started` section has no information about how to clone or download the repository locally. Also, since this milestone is about validation specs, there should be a command included about how to run tests

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/models/user.rb
Comment (aamirkhan2478) [inline app/models/user.rb:1]: - [ ] Please add validation for `posts counter` for now you just validate names only so kindly add validation for `posts counter`

> PostsCounter must be an integer greater than or equal to zero.

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @IsmaelMastronardi, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ 🎉 🎉 ✔️ 

Hi @IsmaelMastronardi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] Tests are passing ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**